memory.ini

```diff
@@ -0,0 +1,12 @@
+[PHP]
+
+; Maximum input variable nesting level
+; https://php.net/max-input-nesting-level
+;max_input_nesting_level = 64
+
+; How many GET/POST/COOKIE input variables may be accepted
+;max_input_vars = 1000
+
+; Maximum amount of memory a script may consume
+; https://php.net/memory-limit
+memory_limit = 2G
```

Comment:
waarom is die mem limit gehardcode? We passen het overal mee via de environment vars toch?